src/main/java/com/depromeet/domain/missionRecord/service/MissionRecordService.java

```diff
@@ -82,6 +83,19 @@ public List<MissionRecordFindResponse> findAllMissionRecord(
         return missionRecords.stream().map(MissionRecordFindResponse::from).toList();
     }
 
+    public Long updateMissionRecord(MissionRecordUpdateRequest request, Long recordId) {
+        MissionRecord missionRecord =
+                missionRecordRepository
+                        .findById(recordId)
+                        .orElseThrow(() -> new CustomException(ErrorCode.MISSION_RECORD_NOT_FOUND));
+
+        validateMissionRecordUserMismatch(
+                missionRecord.getMission(), memberUtil.getCurrentMember());
```

Comment:
MemberUtil에 getCurrentMember 메소드가 호출 될때마다 쿼리를 날려서
의도적으로 상단에서 변수에 할당해놓으면 좋을거같아용
```suggestion
        final Member member = memberUtil.getCurrentMember();
        MissionRecord missionRecord =
                missionRecordRepository
                        .findById(recordId)
                        .orElseThrow(() -> new CustomException(ErrorCode.MISSION_RECORD_NOT_FOUND));

        validateMissionRecordUserMismatch(
                missionRecord.getMission(), member);
```